src/app/core/internal/view/rendering/prims/BaseShapePrim.ts

```diff
@@ -48,6 +50,10 @@ export abstract class BaseShapePrim implements Prim {
         ctx.closePath();
 
         ctx.restore();
+
+        // Reset alpha if we set it
+        if (this.style.alpha !== undefined)
```

Comment:
is this `!==` required because of `number`'s truthiness?

Reply:
Yep  :D